src/api/Authentication.js

```diff
@@ -0,0 +1,24 @@
+export class Authentication{
+  constructor(){}
```

Comment:
***Code Qaulity: Nitpick***
Since all we are doing is is static methods here this constructors are not needed at all here..